asciidoc/courses/llm-fundamentals/modules/1-introduction/lessons/1-neo4j-and-genai/lesson.adoc

```diff
@@ -0,0 +1,76 @@
+= Introduction to Neo4j & GenAI
+:order: 1
+
+In this course, we will cover everything you need to know to use Neo4j effectively with Generative AI.
+You will learn to all about Large Language Models (LLMs), Knowledge Graphs and how they can be used together effectively.
+
+So we start on the same page, let's first cover the basics.
+
+== What is Neo4j?
+
+Neo4j is a graph database management system that allows us to store, manage, and query highly connected data.
+Unlike traditional relational databases, which use tables and rows, Neo4j uses a graph-based model with nodes and relationships.
+This makes it particularly well-suited for representing and querying complex, interconnected data.
+
+
+== What are Knowledge Graphs?
+
+Knowledge Graphs are a specific implementation of a Graph Database, where the underlying data is designed to capture and represent knowledge with a rich semantic context.
```

Comment:
I would probably say that KGs capture and integrate information from many different sources, representing the inherent knowledge of the domain